Control Center Docs Page (#1826)
OctopusDeploy/docs: docs/getting-started/glossary.md

---
title: Glossary
description: A page containing the definitions of terms used in Octopus Deploy.
position: 60
---

> Octopus Deploy is a deployment tool.  It takes the packages and artifacts generated by your build server and deploys them to various targets, be it Windows, Linux, Azure, AWS, or Kubernetes, in a safe and consistent process.

Reviewer: ```suggestion
Octopus Deploy is a deployment tool. It takes your build server's packages and artifacts and deploys them to various targets using a safe and consistent process. Targets you can deploy to include Windows, Linux, Azure, AWS, and Kubernetes.
```
